packages/rspack-plugin-minify/src/index.js

```diff
@@ -24,6 +24,7 @@ module.exports = class RspackMinifyPlugin {
 				target: this.options.target,
 				sourcefile,
 				sourcemap,
+				format: "iife",
```

Comment:
Maybe we should consider the "target" type, see: https://github.com/esbuild-kit/esbuild-loader/commit/a9e8e7e1630ab11371f88f9349cad7ebbce57efc